packages/graphql-transformer-common/package.json

```diff
@@ -36,8 +36,8 @@
     "collectCoverage": true,
     "coverageThreshold": {
       "global": {
-        "branches": 30,
-        "functions": 19,
+        "branches": 14,
```

Comment:
It is a significant coverage change. Can we improve it?

Reply:
Added additional tests to the method I updated and one more similar one. Bumping this number all at once takes more time, will do it incrementally as we update this package.